dm/syncer/safe_mode.go

```diff
@@ -48,11 +48,39 @@ func (s *Syncer) enableSafeModeInitializationPhase(tctx *tcontext.Context) {
 		s.safeMode.Add(tctx, 1) // enable and will revert after pass SafeModeExitLoc
 		s.tctx.L().Info("enable safe-mode for safe mode exit point, will exit at", zap.Stringer("location", *s.checkpoint.SafeModeExitPoint()))
 	} else {
+		initPhaseSeconds := s.cfg.SafeModeDuration
+
+		failpoint.Inject("SafeModeInitPhaseSeconds", func(val failpoint.Value) {
+			seconds, _ := val.(string)
+			initPhaseSeconds = seconds
+			s.tctx.L().Info("set initPhaseSeconds", zap.String("failpoint", "SafeModeInitPhaseSeconds"), zap.String("value", seconds))
+		})
+		var duration time.Duration
+		var err error
+		if initPhaseSeconds == "" {
```

Comment:
Seems already adjust and it could not be empty

Reply:
for some unit test